src/DisabledPluginInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\ManagerPlugin;
+
+interface DisabledPluginInterface
```

Comment:
I think the name `Disabled` is misleading here as implementing this interface doesn’t disable the plugin, but it makes it possible to be disabled by the `isDisabled()` method.

```suggestion
interface DisableablePluginInterface
```

Reply:
Haha, I said the same but @aschempp argued that the other interfaces are the same. It's not an `AlterExtensionInterface` either :)

Reply:
Which other interfaces have a stateful name like “disabled” in their name?

Reply:
What I'm saying is that our current interfaces do not say what you "can do" either. So `Disableable` doesn't work. It should probably be `DisablePluginInterface`.
The others are `RoutingPluginInterface`, `ConfigPluginInterface` etc. None of them is named `ConfigureRoutingPluginInterface` or `InfluenceConfigPluginInterface`.

Reply:
To me “Routing” and “Config” are not about a state or an action whereas “disabled” and “disable” are. We also have several `…able` interfaces in Contao too.

But I agree, `DisablePluginInterface` would be fine for me too. 
Just not “disabled” as this reads to me like a marker interface that it is not.

Reply:
A marker interface is an interface that does not contain methods. Since this one contains methods (to disable a plugin), it obviously is not a marker interface to me. But I agree `DisablePluginInterface` is better, changed in 7ab86eb4d8cac4c8c07573a365ec73916eb6a17a